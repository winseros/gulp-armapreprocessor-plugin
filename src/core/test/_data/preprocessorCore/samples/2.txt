//single line comment
SOME CODE /*multi line comment begins here
commented line
commented line */
SOME CODE
SOME CODE
SOME CODE
SOME CODE
//single line comment
/* multi line comment begins here
commented line
commented line 
*/