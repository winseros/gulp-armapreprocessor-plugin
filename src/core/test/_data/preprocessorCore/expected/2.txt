
SOME CODE 


SOME CODE
SOME CODE
SOME CODE
SOME CODE




